blaze-client/src/test/scala/org/http4s/client/blaze/PooledClientSpec.scala

```diff
@@ -12,14 +13,21 @@ import scala.util.Random
 import org.http4s.client.testroutes.GetRoutes
 import org.http4s.client.JettyScaffold
 
-class MaxConnectionsInPoolSpec extends Http4sSpec {
+import scala.concurrent.Await
```

Comment:
Can we sort all these imports alphabetically?  That's how it's done in most places in the code.  I think this one got IntelliJ'ed.

Reply:
I was relying on scalafmt to do that.